### DISCUSSION TITLE
Dashboard v.3

### DISCUSSION THREAD

**Discussion by EdwyReed** [jgraichen/redmine_dashboard#282] [General]
Hi, @jgraichen!
Me and my team would like to use your plugin for our dashboard. 
But it will be incomparably amazing if plugin will include Agile or other boards. So, we wondering, can you please answer us - how much does it cost (in terms of time and money) to complete third version of the dashboard?

**Comment by jgraichen** [comment]
Hi @EdwyReed,

Thank you for your interest. You are free to use the plugin as you like.

Regarding your question about including "agile" boards, this depends on how do agile and how you intend to implement your agile frameworks. Some users use Redmine versions as sprints and filter for that on the board, some use a single board in Kanban-like fashion. There does not seem to be that one agile board type.

If you have any specific needs, wishes, or ideas for specific workflows you would like to have, you can create issues, explaining your needs. The more details you add, the easier it will be to implement.

For the third version, I have the vision of implementing a more responsive and flexible board with client-site JavaScript. This would support much easier and snappier interaction and more and easier dialogs, such as quickly editing and issue. I am doing this purely in my free spare time, but I am open to any contribution, also to v2. Therefore, I cannot give you an estimation in time or money, as it would either be random, or come with obligations, that I myself am not willing to give, sorry.

I tried a few more interactive features for v3, such as quick edit assignees or custom fields, but these were limited by the internal fields Redmine provides for these things, such as no avatars by default. Therefore, the more recent focus shifted to reimplement the existing features with the following new features:

* Persistent project and global dashboards
* Individual access rules
* Using issues queries for dashboards

These were some of the most wished features in the last years. If you have more wished regarding "agile" boards, "sprints", or other things, please open issues for these. You are welcome to add images and drafts, on how your workflow should be, and maybe, it can be implemented in v3 or later minor.

I hope I could answer most of your questions.

Best regards, @jgraichen
